src/styles/main.scss

```diff
@@ -1 +1,61 @@
 // put code here
+@import "./utils/mixins.scss";
+
+$size-watch: 80vmin;
+$percent-value: 50%;
+
+body {
```

Comment:
Try to specify styling through classes where possible, rather than working with tags directly. For example, you now have a `wrapper` class in HTML, but you don't style it at all.